### PR TITLE
Add dealing with collections, please check it if it correct

### PR DESCRIPTION
**Pull request by glanttz**

### CODE REVIEW

File: src/composables/getCollections.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { ref } from 'vue';
+import { projectFirestore } from '../firebase/config';
+
+const getCollection = (collection) => {
```
Comment (PixelSculptor) [inline src/composables/getCollections.js:4]: Shouldn't be async function? Consider cause you make a request to firestore so this is asynchronous operation.

File: src/composables/getCollections.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    const error = ref(null);
+
+    const collectionRef = projectFirestore
+        .collection(collection)
+        .orderBy('createdAt');
+
+    collectionRef.onSnapshot(
+        (snap) => {
+            const results = [];
+            snap.docs.forEach((doc) => {
```
Comment (PixelSculptor) [inline src/composables/getCollections.js]: I think array.map() would be better function. Save from mutuation.

File: src/composables/getCollections.js (hunk 1 above)
Comment (glanttz) [reply src/composables/getCollections.js:4]: i explain it yesterday, i prefer to not change it.

**Review by PixelSculptor — APPROVED** [review]

File: src/composables/getCollections.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+const getCollection = (collection) => {
+    const documents = ref(null);
+    const error = ref(null);
+
+    const collectionRef = projectFirestore
+        .collection(collection)
+        .orderBy('createdAt');
+
+    collectionRef.onSnapshot(
+        (snap) => {
```
Comment (PixelSculptor) [inline src/composables/getCollections.js:13]: I think we should add new prop: uId to get documents that are belonged to only one patient.

**Pull request merged**